### ISSUE TITLE
[Bug] : keyframe names with e at beginning are trying to convert

### ISSUE THREAD

**Issue by SkReD** [cssnano/cssnano#909]
**Describe the bug**
Keyframes names like those `h1{ -webkit-animation: e836684w2 } h2{ animation: e836684w2 }` with `e` at beginnig are trying to convert by `postcss-convert-values`

**To Reproduce**
Here is PR with failing test on this case
https://github.com/cssnano/cssnano/pull/908

**Expected behavior**
Expected that no conversion performed

**Screenshots**

**Desktop (please complete the following information):**
- CSSNANO Version - `4.1.10`
- preset - `default` 
- Add the output of `npx envinfo && npm ls cssnano` here
```
  System:
    OS: Windows 10 10.0.18363
    CPU: (12) x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
    Memory: 4.04 GB / 15.86 GB
  Binaries:
    Node: 12.14.1 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.19.1 - D:\Soft\Yarn\bin\yarn.CMD
    npm: 6.10.2 - C:\Program Files\nodejs\npm.CMD
  Managers:
    Cargo: 1.31.0 - ~\.cargo\bin\cargo.EXE
    pip2: 19.2.3 - d:\soft\python27\Scripts\pip2.EXE
  Utilities:
    Git: 2.24.0.
  Virtualization:
    Docker: 19.03.8 - C:\Program Files\Docker\Docker\resources\bin\docker.EXE
  Languages:
    Bash: 4.4.19 - C:\Windows\system32\bash.EXE
    Go: 1.11.2 - D:\Soft\Go\bin\go.EXE
    Python: 2.7.17
  Databases:
    MongoDB: 4.0.1 - D:\Soft\MongoDB\Server\4.0\bin\mongo.EXE
  Browsers:
    Edge: 44.18362.449.0
    Internet Explorer: 11.0.18362.1
  Monorepos:
    Yarn Workspaces: 1.19.1

fb-mobile@1.281.0 C:\CSSSR\mvno-forms
`-- @tinkoff-boxy/mmf@1.51.1
  `-- @tinkoff-boxy/mmb@3.37.0
    `-- optimize-css-assets-webpack-plugin@5.0.3
      `-- cssnano@4.1.10
```

**Comment by anikethsaha** [comment]
I can confirm this is a bug and I can [reproduce it](https://cssnano.netlify.app/playground#eyJpbnB1dCI6Ii8qIHdyaXRlIHlvdXIgY3NzIGJlbG93ICovXHJcblxyXG5oMSB7XHJcbiAgLXdlYmtpdC1hbmltYXRpb246IGU4MzY2ODR3MjtcclxufVxyXG5oMiB7XHJcbiAgYW5pbWF0aW9uOiBlODM2Njg0dzI7XHJcbn1cclxuIiwiY29uZmlnIjoiLy8gY3NzbmFubyBjb25maWdcbntcbiAgXCJwcmVzZXRcIiA6IFwiZGVmYXVsdFwiLFxufVxuIn0=).


we need to ignore transformation for `animation` and others which expects custom names

**Comment by SkReD** [comment]
Looks like this was fixed here https://github.com/TrySound/postcss-value-parser/commit/7867d1f1c18cee4bccd0b582ff7e27618148a865

**Comment by anikethsaha** [comment]
we need to update `postcss-value-parser` to `v4.1.x`. [ref this](https://github.com/cssnano/cssnano/blob/master/packages/postcss-convert-values/package.json#L31)

Feel free to submit a PR

**Comment by blephy** [comment]
I'm also running on a similar issue with webpack / css module : 

When using css module features, and using @keyframes, some keyframes name produce an invalid hash in the related css animation name.

```scss
.signature {
  width: 80px;

  path {
    animation: draw 3.3s ease-in-out forwards;
    stroke: #fff;
    stroke-dasharray: 450;
    stroke-dashoffset: 450;
  }

  circle {
    animation: draw-circle 0.25s ease-in 1.5s forwards;
  }
}

@keyframes draw {
  to {
    stroke-dashoffset: 0;
  }
}

@keyframes draw-circle {
  from {
    fill: #fff;
    opacity: 0;
  }

  to {
    fill: #fff;
    opacity: 1;
  }
}
```

is producing this :

```css
._2f-763U9aR_m95TZkSTerd{width:80px}._2f-763U9aR_m95TZkSTerd path{animation:NaNjtcXwnmOr8f36q3l-8W 3.3s ease-in-out forwards;stroke:#fff;stroke-dasharray:450;stroke-dashoffset:450}._2f-763U9aR_m95TZkSTerd circle{animation:_2APW8yWcYXdDHglJ5UTsYC .25s ease-in 1.5s forwards}@keyframes e5jtcXwnmOr8f36q3l-8W{to{stroke-dashoffset:0}}@keyframes _2APW8yWcYXdDHglJ5UTsYC{0%{fill:#fff;opacity:0}to{fill:#fff;opacity:1}}
```

Note the `animation:NaNjtcXwnmOr8f36q3l-8W`. Should be `animation:e5jtcXwnmOr8f36q3l-8W`.

- When i do not use `css-minimizer-webpack-plugin`, the bug disapear. Which use `cssnano`

- When i rename the keyframe's name from `draw` to `draw-path` for exemple, this is working well with `css-minimizer-webpack-plugin`.

You can reproduce by following this steps in this repo :

1. [Follow this steps](https://github.com/blephy/reactjs-typescript-pwa#use-svg-files) to add the `src/images/signature.svg` to [home view](https://github.com/blephy/reactjs-typescript-pwa/blob/master/src/views/home/index.tsx)
2. Just make a production build : `npm run build`
3. Launch the production server: `npm run start:prod`

See build/css/home.css result. Webpack configuration are in `config/webpack`

- Operating System: MacOSX Catalina 10.15.5
- Node Version: 12.18.3
- NPM Version: 6.14.8
- webpack Version: 4.44.2
- css-minimizer-webpack-plugin Version: 1.1.4

Issue already asked in `css-loader` repository : https://github.com/webpack-contrib/css-loader/issues/1200 and webpack repository : https://github.com/webpack/webpack/issues/11568 and `css-minimizer-webpack-plugin` https://github.com/webpack-contrib/css-minimizer-webpack-plugin/issues/39 🤣

**Comment by ludofischer** [comment]
Fixed in 5.0.0-rc.2
